content/scalability/docs/index.md

```diff
@@ -0,0 +1,28 @@
+
+## How to use this guide
+This guide is meant for developers and administrators responsible for creating and managing EKS clusters in AWS. It focuses on some generic Kubernetes scaling practices, but it does not have specifics for self-managed Kubernetes clusters or clusters that run outside of an AWS region with [EKS Anywhere](https://anywhere.eks.amazonaws.com/).
+
+Each topic has a brief overview, followed by recommendations and best practices for operating EKS clusters at scale. Topics do not need to be read in a particular order and recommendations should not be applied without testing and verifying they work in your clusters.
+
+## Understanding scaling dimensions
+Scalability is different from performance and [reliability](https://aws.github.io/aws-eks-best-practices/reliability/docs/), and all three should be considered when planning your cluster and workload needs. As clusters scale, they need to be monitored, but this guide will not cover monitoring best practices. EKS can scale to large sizes, but you will need to plan how you are going to scale a cluster beyond 300 nodes or 5000 pods. These are not absolute numbers, but they come from collaborating this guide with multiple users, engineers, and support professionals.
+
+Scaling in Kubernetes is multi-dimensional and there are no specific settings or recommendations that work in every situation. The main areas areas where we can provide guidance for scaling include:
+
+* [Kubernetes Control Plane](control-plane)
+* [Supporting Infrastructure](supporting-infrastructure)
+* [Cluster Services](cluster-services)
+* [Workloads](workloads)
+
+**Kubernetes Control Plane** in an EKS cluster includes all of the services AWS runs and scales for you automatically (e.g. Kubernetes API server). Scaling the Control Plane is AWS's responsibility, but using the Control Plane responsibly is your responsibility.
+
+**Supporting infrastructure** scaling deals with AWS resources that are required for your cluster and workloads, but they are outside of the EKS Control Plane. Resources such as EC2—including kublet, load balancers, and storage should all be scaled as your cluster scales.
+
+**Cluster services** are Kubernetes controllers and applications that run inside the cluster and provide functionality for your cluster and workloads. These can be [EKS Add-ons](https://docs.aws.amazon.com/eks/latest/userguide/eks-add-ons.html) and also other services or Helm charts you install for compliance and integrations. These services are often depended on by workloads and as your workloads scale your cluster services will need to scale with them.
+
+**Workloads** are the reason you have a cluster and should scale horizontally with the cluster. There are integrations and settings that workloads have in Kubernetes that can help the cluster scale. There are also architectural considerations with Kubernetes abstractions such as namespaces and services.
+
+## Feedback
```

Comment:
This is covered in the main guide. I'm not sure we need to add it here.

Reply:
I saw other pages had it eg https://aws.github.io/aws-eks-best-practices/security/docs/ which is why I added it. If we should remove it that's fine too.

Reply:
I should probably remove that. I think I had it in there when it was the first and only module.